### PR TITLE
ADD: Parallax image, lesson tracking service, upgrade angular version 8 -> 9

### PR DESCRIPTION
**Pull request by mfish33**
Trello links:
https://trello.com/c/2L3HE3MA/47-begin-course-progress-tracking
https://trello.com/c/CAlrOqXX/46-clean-up-css-files-colors-fonts-etc

### CODE REVIEW

File: src/app/core/navbar/navbar.component.html [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+                <style>
+                    .cls-1 {
+                        fill: #427AA1;
+                    }
+
+                    .cls-2 {
+                        fill: #6EA0C4;
+                    }
+                </style>
+            </defs>
```
Comment (kaitrias) [inline src/app/core/navbar/navbar.component.html:22]: Is there a reason why you put the css here?

File: src/styles.scss [hunk 2]
```diff
@@ -21,4 +22,51 @@ html, body {
...
+    font-size: 115px;
+    line-height: 124px;
+}
+
+.frank-small-header{
+    @extend .frank;
+    font-weight: bold;
+    font-size: 64px;
+    line-height: 83px;
+}
```
Comment (kaitrias) [inline src/styles.scss:46]: thoughts on specifying margins for our styling?

File: src/app/core/navbar/navbar.component.html (hunk 1 above)
Comment (mfish33) [reply src/app/core/navbar/navbar.component.html:22]: This is your SVG from the hackathon. The diff is from me saving the file and my autoFormatter. That is a good point though and I'll go put it in the style sheet.

File: src/styles.scss (hunk 2 above)
Comment (mfish33) [reply src/styles.scss:46]: The default margin seems to be good in most cases except for headings so we can maybe do something about that. Do you have any ideas for default values?

**Pull request merged**